views/userpageexternal.hbs

```diff
@@ -0,0 +1,200 @@
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script src="https://momentjs.com/downloads/moment.js"></script>
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
+
+    <link rel="stylesheet" href="/stylesheets/userpagestyle.css">
+    <link rel="stylesheet" href="/stylesheets/standardstyle.css">
+  </head>
+  <body>
+    <nav class="navbar navbar-inverse navbar-fixed-top navbar-custom navbar-custom" role="navigation">
+    <div class="container">
+      <!-- Deadlock logo -->
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>                        
+        </button>
+        <a class="navbar-brand" href="/"><img src="/images/" width="100%" height="100%" class="d-inline-block align-top" alt="Deadlock"></a>
+        <!-- <a href="/"><img src="images/lock.png" style="margin:5px; width: 20%;height: 20%;" ></a>   --> 
+      </div>
+
+      <div class="collapse navbar-collapse" id="myNavbar">
+        <ul class="nav navbar-nav">
+          <li class="nav-item"><a style="color:white" href="/">Home</a></li>
+          {{#if oguser}}
+            <li class="nav-item"><a style="color:orange" href="/loginRegister/logout">Logout</a></li>
+          {{else}}
+            <li class="nav-item"><a style="color:white" href="/loginRegister">Log in / Register</a></li>
+          {{/if}}
+          <li class="nav-item"><a style="color:white" href="/aboutus">About Us</a></li>
+          <li class="nav-item"><a style="color:white" href="/userpage">User Page</a></li> 
+          <li class="nav-item"><a style="color:white" href="/jobPage">View Job List</a></li>  
+          {{#if user.admin}}  
+            <li class="nav-item"><a style="color:white" href="/jobPostingPage">Post A Job</a></li>
+          {{/if}}
+          {{#if user.company}}  
+            <li class="nav-item"><a style="color:white" href="/jobPostingPage">Post A Job</a></li>
+            <li class="nav-item"><a style="color:white" href="/userRankPage">User Rankings</a></li>
+          {{/if}}
+          {{#if user.admin}}  
+            <li class="nav-item"><a style="color:white" href="/companyAccount">Create Company Account</a></li>
+          {{/if}}    
+        </ul>              
+      </div>
+    </div>
+  </nav>
+
+    <div class="jumbotron text-center" style="background-color: #116CF6; margin-top: 50px">
```

Comment:
The rest of this seems to be copied from `userpage.hbs` - why is this code copied? As above - couldn't you modify `userpage.hbs` directly?